docs/howto/hooks/lua.md

```diff
@@ -703,14 +739,16 @@ Parameters:
 - `glue`: AWS glue client
 - `db(string)`: glue database name
 - `table_src_path(string)`: path to table spec (e.g. _lakefs_tables/my_table.yaml)
-- `create_table_input(Table)`: Input equal mapping to [table_input](https://docs.aws.amazon.com/glue/latest/webapi/API_CreateTable.html#API_CreateTable_RequestSyntax) in AWS, the same as we use for `glue.create_table`.
+- `create_table_input(table)`: Input equal mapping to [table_input](https://docs.aws.amazon.com/glue/latest/webapi/API_CreateTable.html#API_CreateTable_RequestSyntax) in AWS, the same as we use for `glue.create_table`.
 should contain inputs describing the data format (e.g. InputFormat, OutputFormat, SerdeInfo) since the exporter is agnostic to this. 
 by default this function will configure table location and schema.
-- `action_info(Table)`: the global action object.
-- `options(Table)`:
+- `action_info(table)`: the global action object.
+- `options(table)`:
   - `table_name(string)`: Override default glue table name
   - `debug(boolean`
   - `export_base_uri(string)`: Override the default prefix in S3 for symlink location e.g. s3://other-bucket/path/
+  - `create_db_input(table)`: if this is specified, then it indicates we want to create a new database for the table export. The parameter expects a table that is converted to JSON and passed "as is" to AWS and is parallel to the AWS SDK [CreateDatabaseInput](https://docs.aws.amazon.com/glue/latest/webapi/API_CreateDatabase.html#API_CreateDatabase_RequestSyntax)
+  - `error_on_db_exists(boolean)`: Relevant when create_db_input is specified. This parameter specifies how to handle the case where we try to create a DB but one already exists- should the exporter throw an error or just use the existing database?
```

Comment:
I spoke to @talSofer , she doesn't think this flag is needed on the high level export function, since we know of no usecase for this. It should ways be "create or get" when dbinput is specified